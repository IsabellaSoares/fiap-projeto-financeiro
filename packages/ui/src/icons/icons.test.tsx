import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";
import { Icons } from ".";

describe("Icons", () => {
  it("will check icon for no changes", () => {
    const { container } = render(<Icons name="mdiRocketLaunch" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
