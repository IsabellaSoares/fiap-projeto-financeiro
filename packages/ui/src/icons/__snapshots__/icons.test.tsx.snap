// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Icons > will check icon for no changes 1`] = `<span />`;
